lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -280,12 +317,21 @@ def _configure_nodes(self, environment: Environment, log: Logger) -> None:
             node_context.cloud_init_file_path = os.path.join(
                 vm_disks_dir, f"{node_context.vm_name}-cloud-init.iso"
             )
-            node_context.os_disk_base_file_path = qemu_node_runbook.qcow2
+
+            if self.host_node.is_remote:
```

Comment:
1. the `is_remote` is incorrect here, and always be True, because it's the method itself. The `is_remote()` is right.
2. I'm not sure if the "remote" check is useful here. Can remote and local use the same folder structure?

Reply:
1. `is_remote` is correct based on usages in other files. `is_remote()` throws the error "bool is not callable".
2. This is an optimization. We avoid copying OS disk image for local node. That's why remote check is useful here.

Reply:
`is_remote` is marked with `@property`. So, it behaves like a C# style property.

Reply:
Sorry, I'm confused on the LibVirtHost, I thought its type is LibVirtHost. BTW, what's the reason it needs a LibVirtHost schema? Can it reuse the RemoteNode schema?

Reply:
I think there is some utility in having a separate schema because we could have libvirt specific properties in it. Right now we have the `lisa_working_dir` property that is not part of `RemoteNode`. In the future we could have a `supported_hypervisors` property for example to indicate the hypervisors (qemu, ch etc) that the host supports.

Reply:
The Qemu can inherit the RemoteNode, so it's easy to obtain the general supports of node. The Node has working_path, and it can be overwritten to read from the schema.